services/core-web/src/components/mine/Incidents/MineIncident.js

```diff
@@ -0,0 +1,319 @@
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { flattenObject } from "@common/utils/helpers";
+import { connect } from "react-redux";
+import { Link, withRouter } from "react-router-dom";
+import {
+  change,
+  submit,
+  getFormSyncErrors,
+  getFormValues,
+  reset,
+  touch,
+  isDirty,
+} from "redux-form";
+import { Tag } from "antd";
+import { ArrowLeftOutlined, EnvironmentOutlined } from "@ant-design/icons";
+import PropTypes from "prop-types";
+import moment from "moment";
+import { getMineIncident } from "@common/reducers/incidentReducer";
+import {
+  createMineIncident,
+  fetchMineIncident,
+  updateMineIncident,
+  removeDocumentFromMineIncident,
+} from "@common/actionCreators/incidentActionCreator";
+import { clearMineIncident } from "@common/actions/incidentActions";
+import AuthorizationGuard from "@/HOC/AuthorizationGuard";
+import * as FORM from "@/constants/forms";
+import * as Permission from "@/constants/permissions";
+import Loading from "@/components/common/Loading";
+import customPropTypes from "@/customPropTypes";
+import IncidentForm from "@/components/Forms/incidents/IncidentForm";
+import ScrollSideMenu from "@/components/common/ScrollSideMenu";
+import * as routes from "@/constants/routes";
+
+const propTypes = {
+  // eslint-disable-next-line react/no-unused-prop-types
+  incident: customPropTypes.incident.isRequired,
```

Comment:
Is this ignore still needed cause I can see this prop being used.

Reply:
This is a known bug with ESLint where it complains it's not being used when it is, we've had to add this in several areas throughout the project because of "reasons".